### ISSUE TITLE
My application don't  run because of an error with merge-anything

### ISSUE THREAD

**Issue by Nathan-html** [acro5piano/react-native-big-calendar#870]
Hi,
I am using react-native-web in my TS expo application.

and when I run my project with the Getting Started code I got the error bellow

```
ERROR in ./node_modules/react-native-big-calendar/build/index.es.js:17:14
Module not found: Package path ./dist/index.es is not exported from package ***\app\node_modules\react-native-big-calendar\node_modules\merge-anything (see exports field in ***\app\node_modules\react-native-big-calendar\node_modules\merge-anything\package.json)
  15 | // It is still possible to use .cjs by changing Metro config, but it forces library users to 
take an additional step.
  16 | // So this workaround is better.
> 17 | const merge = require('merge-anything/dist/index.es').merge as typeof TMerge
```

I add this code to my metro.config.json
```
// Learn more https://docs.expo.io/guides/customizing-metro
const { getDefaultConfig } = require('expo/metro-config');

const config = getDefaultConfig(__dirname);

config.resolver.assetExts.push('jsx');
config.resolver.assetExts.push('js');
config.resolver.assetExts.push('tsx');
config.resolver.assetExts.push('ts');
config.resolver.assetExts.push('cjs');
config.resolver.assetExts.push('svg');

module.exports = config;
```

and I try to run my app with merge-anything downloaded but it still don't work.

Do you know how can I fix it ?

**Comment by acro5piano** [comment]
Duplicated as this. I'll fix it soon, by implementing our own `merge` function.

https://github.com/acro5piano/react-native-big-calendar/issues/851

**Comment by acro5piano** [comment]
Fixed on v3.4.2. I removed `merge-anything` module completely. If you are using the hacky patch-package workaround, please stop using it and try the latest version.
